Dnn.AdminExperience/EditBar/Dnn.EditBar.UI/editBar/css/theme.css

```diff
@@ -0,0 +1,9 @@
+/*--------------------------------------------------------------------------------
+These default CSS variables can be overridden and customized by creating a file, 
+containing a copy of the below, called "EditBarTheme.css" and placing it in the 
+"Portals/_default" directory. If the custom CSS file exists, it will be loaded 
+instead of this one.
+---------------------------------------------------------------------------------*/
+:root{
+    --dnn-color-editbar-background: #0b1c24;
```

Comment:
If we are adding an ability to customize with variables, should we expand and allow other changes this way as well?  Foreground color etc

Reply:
Yes, this has great potential for future expansion and greater customization.  This simply establishes the initial base styling goals.